### DISCUSSION TITLE
doris创建oracle catalog查询数据报unkown table

### DISCUSSION THREAD

**Discussion by zhouzonglisrj** [apache/doris#21358] [A - General / Q&A]
doris创建oracle catalog，show tables能查询出来oracle数据库所有的表名，但是查询某个数据表的数据报unkown table

**Comment by yagagagaga** [comment]
Has any EXCEPTIONs in fe.log ?

**Comment by caoliang-web** [comment]
create catalog ，you can specify lower_case_table_names = true try
